distributed/scheduler.py

```diff
@@ -1333,7 +1330,9 @@ def __init__(
             self.periodic_callbacks["idle-timeout"] = pc
 
         if extensions is None:
-            extensions = DEFAULT_EXTENSIONS
+            extensions = list(DEFAULT_EXTENSIONS)
```

Comment:
Copied to avoid modifying the `DEFAULT_EXTENSIONS` by accident here.

Reply:
Ah yes, good catch